src/Standards/Generic/Tests/ControlStructures/ControlStructuresBracketsUnitTest.inc

```diff
@@ -0,0 +1,170 @@
+<?php
```

Comment:
The test file doesn't have any examples of comments being placed between the closing parenthesis and the opening brace of the control structures, so it's not obvious how those would be fixed. A few examples of using `// comment` and the phpcs annotations like `// phpcs:ignore -- for reasons` would be good, just to make sure the sniff copes with them on other people's code.

Reply:
I'll see about adding those cases